lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1113,7 +1113,7 @@ def compute_severity(record, entry)
       elsif record.key?('severity')
         return parse_severity(record.delete('severity'))
       elsif @service_name == CONTAINER_SERVICE && \
-            entry.metadata.labels.key?("#{CONTAINER_SERVICE}/stream")
+            entry.metadata.labels.has_key?("#{CONTAINER_SERVICE}/stream")
```

Comment:
Does it make sense to file a bug with the protobuf folks to add the `key?` method to proto maps?